atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +151,25 @@ def set_media_types(self, extra, worker_metadatas):
                 extra['image']['media_types'] = json.loads(annotations['media-types'])
                 return
 
+    def set_manifest_list_info(self, extra, worker_metadatas):
+        manifest_list_digests = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
+        # group_manifests returns either a list of manifest digests of the manifest lists,
+        # or a list of the image_manifests
+        if manifest_list_digests and not isinstance(manifest_list_digests[0], dict):
+            index = {}
+            index['tags'] = [image.tag for image in self.workflow.tag_conf.images]
+            repositories = self.workflow.build_result.annotations['repositories']['unique']
+            repo = ImageName.parse(repositories[0]).repo
+            # group_manifests added the registry, so this should be valid
+            for registry in self.workflow.push_conf.docker_registries:
```

Comment:
If Pulp is used, we only want to include the Pulp registry. Otherwise, we want to include all registries we pushed to. So it should be:

```
registries = self.workflow.push_conf.pulp_registries
if not registries:
    registries = self.workflow.push_conf.all_registries
for registry in registry:
```